userguide/dailyuse/anbox.rst

```diff
@@ -27,37 +28,61 @@ How to install
     Because this feature is in such an early stage of development, the installation is only recommended for experienced users.
 
 - :doc:`Install <../install>` the 16.04/devel channel on your supported device
-- Open a terminal and run ``export CODENAME="turbo" && export PARTITIONNAME="bootimg"``, but replace the part between the quotes respectively with the codename and name of the boot partition for your device. See the above list.
 - Activate developer mode on your device.
 - Connect the device to your computer computer and run the following commands::
 
-    adb shell sudo reboot -f bootloader
+    adb shell 
+    sudo reboot bootloader
     wget http://cdimage.ubports.com/anbox-images/anbox-boot-$CODENAME.img
     sudo fastboot flash $PARTITIONNAME anbox-boot-$CODENAME.img
     sudo fastboot reboot
     rm anbox-boot-$CODENAME.img
 
-- wait for the device to reboot, then run::
+- Wait for the device to reboot, then run::
 
     adb shell
     sudo mount -o rw,remount /
     sudo apt update
     sudo apt install anbox-ubuntu-touch
-    anbox-setup
-    exit
+    anbox-tool install
 
-- Done! Select "Anbox" in the apps scope to use android applications. You might have to refresh the apps scope (pull down from the center of the screen and release) for the app to show up.
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the app to show up.
 
 How to install new apks
 -----------------------
 
-- Copy the apk to `/home/phablet/anbox-data/data`
-- Login to android container on your phone
+- Copy the apk to `/home/phablet/Downloads`::
 
-    lxc-console -ndefault -P /home/phablet/anbox-data/containers -t0
+   adb shell
+   sudo mount -o rw,remount /
+   sudo apt update
+   sudo apt install android-tools-adb
+   adb install my-app.apk
 
-- cd into `data/` 
-- install the apk by using `pm install [appname].apk`
+- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the app to show up.
+
+Keep your apps up to date
+-------------------------
+
+- To keep your apps running the lasted version you can make use off::
+
+     ApkTrack.
+     Checks, if updates for installed apps are available.
+     You can found ApkTrack on GitHub.
+
+How to uninstall applications
+-----------------------------
+
+- How to uninstall apps::
+
+   adb shell
+   sudo mount -o rw,remount /
+   anbox-tool app-list
+   this is a example of the app-list installed apps on your device:
+   ApkTrack (fr.kwiatkowski.ApkTrack)
+   adb uninstall fr.kwiatkowski.ApkTrack
```

Comment:
Also change the reference to ApkTrack here

Reply:
I will fix this